scripts/schema/config/model.json

```diff
@@ -10,8 +10,8 @@
     "L0": { "type": "number", "exclusiveMinimum": 0.0 },
     "Lc": { "type": "number", "exclusiveMinimum": 0.0 },
     "Partition": { "type": "string" },
-    "ReorientTetMesh": { "type": "boolean" },
     "RemoveCurvature": { "type": "boolean" },
+    "ReorientTetMesh": { "type": "boolean" },
```

Comment:
Ideally the order here mirrors the order in `configfile.cpp` so these are easily to keep track of, so this can be reverted.